src/components/Alerta.js

```diff
@@ -41,7 +43,7 @@ export default function Alerta({ alerta, id, checkados, setCheckados }, index) {
       </TableCell>
 
       <TableCell align="center" className={classes.level}>
-        {alerta.level}
+        <Alert style={{ textAlign: "center" }} severity={alerta.level === 'debug' ? 'info' : alerta.level}>{alerta.level}</Alert>
```

Comment:
```suggestion
        <Alert style={{ textAlign: "center" }} severity={alerta.level === 'debug' ? 'info' : alerta.level}>{alerta.level.toUpperCase()}</Alert>
```

Se fizermos essa modificação o texto da label sempre ficará em maiusculo, assim como na pagina de descrição do alerta.

Reply:
Feito